doc/tutorial/Server.lhs

```diff
@@ -8,7 +8,15 @@ type. Can we have a webservice already?
 Equipped with some basic knowledge about the way we represent APIs, let's now
 write our first webservice.
 
-The source for this tutorial section is a literate haskell file, so first we
+## Setting up our project
+
+Don't forget to look at the [cabal file included with this example project](doc/tutorial/tutorial.cabal). You can run the main function provided in the following examples within the `Server` module without using a separate executable.
+
+If you're using Stack, then in your `stack.yaml` you may want to specify version 0.11.1.1 for `aeson`, and version 0.5 for `servant` and `servant-server` in your `extra-deps`.
```

Comment:
@soenkehahn I think most people (myself included) will setup a new project and type things in from that without necessarily knowing where the `.lhs` files live.

Good feedback, I'll give it another pass when I have time again.